src/include/kernel/abi.h

```diff
@@ -30,7 +30,7 @@
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 27
-#define SOF_ABI_PATCH 0
+#define SOF_ABI_PATCH 1
```

Comment:
Is this a bug fix ?  I prefer a minor change

Reply:
As commented in https://github.com/thesofproject/linux/pull/4475/commits/0876dffe3d39b9b252cf3e1ba072e63e4edbc2f6#r1273099107 - in principle this change is purely in the software, it doesn't actually change any run-time behaviour, so actually no ABI version change should be needed

Reply:
ack, these IDs were never used so I would be safe to drop the ABI update.

Reply:
Got it, i will force skip the patch change check. @plbossart FYI for when I revert this on the kernel PR as well.